bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableRegionLocator.java

```diff
@@ -54,12 +54,13 @@
 
   protected final TableName tableName;
   private ListenableFuture<List<HRegionLocation>> regionsFuture;
-  private final BigtableDataClient client;
+  private final BigtableDataClientWrapper client;
```

Comment:
Should we use `IBigtableDataClient` here(and everywhere else) instead of using the `BigtableDataClientWrapper`?

Reply:
@rahulKQL, good catch.  I replaced `BigtableDataClientWrapper` where ever I could with `IBigtableDataClient`.